src/aarch64/Ginit.c

```diff
@@ -217,7 +217,7 @@ get_static_proc_name (unw_addr_space_t as, unw_word_t ip,
                       char *buf, size_t buf_len, unw_word_t *offp,
                       void *arg)
 {
-  return _Uelf64_get_proc_name (as, getpid (), ip, buf, buf_len, offp);
+  return _Uelf64_get_proc_name (as, getpid (), ip, buf, buf_len, offp,arg);
```

Comment:
Should be a space after the `,` and before `arg`. Similarly with several other changed files.